worker/codalabworker/main.py

```diff
@@ -112,19 +112,23 @@ def main():
     logger.info('Connecting to %s' % args.server)
     if args.password_file:
         if os.stat(args.password_file).st_mode & (stat.S_IRWXG | stat.S_IRWXO):
-            print >>sys.stderr, """
+            print(
+                """
 Permissions on password file are too lax.
 Only the user should be allowed to access the file.
 On Linux, run:
-chmod 600 %s""" % args.password_file
+chmod 600 %s"""
+                % args.password_file,
+                file=sys.stderr,
+            )
             exit(1)
         with open(args.password_file) as f:
             username = f.readline().strip()
             password = f.readline().strip()
     else:
         username = os.environ.get('CODALAB_USERNAME')
         if username is None:
-            username = raw_input('Username: ')
+            username = eval(input('Username: '))
```

Comment:
2to3 replaced raw_input with eval(input(...)), but perhaps it's safer if we just keep it at input(...)?